bootstrap/standalone/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/standalone/StandaloneMain.java

```diff
@@ -100,7 +100,9 @@ private static void createSession() throws SessionCreationException, SessionUpda
             try {
                 // Update the session
                 sessionManager.updateSession(sessionInfo);
-                sessionManager.logger().info("Updated session!");
+                if(!config.suppressSessionUpdateInfo()){
+                    sessionManager.logger().info("Updated session!");
```

Comment:
Can we make it print this into debug when suppressed, as it's useful for debugging